File: C/1017 - Gasto de Combustível.c
/* 
   ==UserScript==
 @name         Gasto de Combustível
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1017 - Gasto de Combustível
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

#include <stdio.h>

int main()
{
  int Tempo_Gasto, Velocidade_Media;
  double Distancia, Litros_Necessarios;
  
  scanf("%d%d", &Tempo_Gasto, &Velocidade_Media);
  
  Distancia = Velocidade_Media * Tempo_Gasto;
  
  Litros_Necessarios = Distancia/12;
  
  printf("%.3lf\n", Litros_Necessarios);
  
    return 0;
}

File: Java/1017 - Gasto de Combustível.java
/* 
   ==UserScript==
 @name         Distância
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1017 - Gasto de Combustível
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

import java.util.Scanner;
import java.io.IOException;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scan = new Scanner(System.in);
        int tempo = scan.nextInt();
        int velocidadeMedia = scan.nextInt();
        double distancia = velocidadeMedia * tempo;
        
        double litrosNecessarios = distancia/12;
        
        System.out.println(String.format("%.3f", litrosNecessarios));
        
    }
 
}
